### ISSUE TITLE
Tournaments results not being calculated correctly

### ISSUE THREAD

**Issue by lachlan-t** [Kunfury-blep/Blep-Fishing#27]
I've set up a tournament for the longest fish of any type.
During the tournament, we get notified when someone takes the lead, as is expected.

However, when the tournament finishes, it says that no fish were caught, and then assigns two players to 1st and 2nd (not 3rd) place and gives them the rewards.

**Comment by Kunfury-blep** [comment]
Yup I've been looking deeply into this after other users reported much of the same. I'm in the process of reworking the entire system and it should work properly and more efficiently than before!

Thank you for reporting!
